### PR TITLE
Update extensionsGallery.json for mongodb

### PR DESCRIPTION
**Pull request by erinstellato-ms**
Need to update the extension gallery with the latest extension from MongoDB (v 1.2.1)

### CODE REVIEW

File: extensionsGallery.json [hunk 1]
```diff
@@ -4776,34 +4776,34 @@
 					},
 					"versions": [
 						{
-							"version": "0.9.3",
-							"lastUpdated": "04/26/2022",
+							"version": "1.2.1",
```
Comment (Charles-Gagnon) [inline extensionsGallery.json:4779]: Update the insiders gallery too

Comment (Charles-Gagnon) [reply extensionsGallery.json:4779]: Nevermind, I went ahead and did it for you 😄

File: extensionsGallery-insider.json [hunk 2]
```diff
@@ -4827,7 +4827,7 @@
 						}
 					],
 					"statistics": [],
-					"flags": "preview"
+					"flags": ""
```
Comment (Charles-Gagnon) [inline extensionsGallery-insider.json:4830]: @erinstellato-ms They removed the preview flag from their package.json, so I removed it here too. That work for you, or would you rather we keep it for the ADS version for now?

(note that there's likely to be some small issues with how that's displayed due to different content in gallery vs extension itself if we do keep it. I can go over that separately with you if needed)

**Review by Charles-Gagnon — CHANGES_REQUESTED** [review]
Blocking until preview flag status is verified

File: extensionsGallery-insider.json (hunk 2 above)
Comment (erinstellato-ms) [reply extensionsGallery-insider.json:4830]: @Charles-Gagnon Checked VS Code and the extension is NOT in preview there, so it looks like we should remove that.

**Review by erinstellato-ms — COMMENTED** [review]
preview flag should be removed

**Pull request merged**